Google-sandbox-best-practices-2026.html

```
@@ -0,0 +1,2343 @@
+        TMPL["SandboxTemplate<br/>セキュリティ設計図"]
+        POOL["SandboxWarmPool<br/>事前起動プール（常時N個待機）"]
+        SB["Sandbox Pod<br/>（gVisor RuntimeClass）"]
+    end
+
+    subgraph STORAGE["Pod Snapshot（GCS）"]
+        SNAP["スナップショット<br/>実行状態を保存・復元"]
+    end
+
+    CLAIM -->|"1. テンプレート参照"| TMPL
+    TMPL -->|"2. 事前起動"| POOL
+    CLAIM -->|"3. Poolから即割り当て（<1秒）"| SB
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)